tgui/packages/tgui/interfaces/PreferencesMenu/antagonists/antagonists/space_sapper.ts

```diff
@@ -0,0 +1,16 @@
+import { Antagonist, Category } from '../base';
+
+const SpaceSapper: Antagonist = {
+  key: 'spacesapper',
+  name: 'Space Sapper',
+  description: [
+    `
+    You're an illegal credits miner! Construct elaborate bases and
+    defend your shuttle and its credit-miner machine at all costs!
+    Or perhaps work together with the station, crime is optional...
```

Comment:
These are generally written with more flavor in mind - compare this to traitor's:
```
      An unpaid debt. A score to be settled. Maybe you were just in the wrong
      place at the wrong time. Whatever the reasons, you were selected to
      infiltrate Space Station 13.
```